addons/binding/org.openhab.binding.freemobilesms/ESH-INF/i18n/freemobilesms_xx_XX.properties

```diff
@@ -0,0 +1,13 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
+# FIXME: please do not add the file to the repo if you add or change no content
+# binding
+binding.freemobilesms.name = <Your localized Binding name>
```

Comment:
Please remove this file or fill in those values for at least one other language

Reply:
I've revised the labels and instanciated the file for French.